src/Demo/suggestionsStore.ts

```diff
@@ -1,41 +1,24 @@
-import { SuggestionMeta } from "../ApiModules/autocompleteSession";
+import { Snippet } from "../ApiModules/autocompleteSession";
 
-const INITIAL_SUGGESTIONS: SuggestionMeta[] = [
-  { suggestion: 'Nice to meet you!', type: 'default', baseClass: 'suggestion', expandedSuggestion: '', userAdded: false },
-  { suggestion: 'Nice to hear from you.', type: 'default', baseClass: 'suggestion', expandedSuggestion: '', userAdded: false },
-  { suggestion: 'What do you think?', type: 'default', baseClass: 'suggestion', expandedSuggestion: '', userAdded: false }
-];
-
-const INITIAL_HOTKEYS: SuggestionMeta[] = [
-  { suggestion: 'about', type: 'custom', baseClass: 'hotkey', expandedSuggestion: "Point uses AI to predict what you want to say and writes it in your own words. With our suggestions, you will communicate quickly, intelligently, and effortlessly.", userAdded: false },
```

Comment:
Can we keep this one exactly the way it was? I'm  not sure but it might be tightly coupled with our tutorial 1.0

Reply:
all refs to suggestionMeta are gone :^(. Tutorial v1 is essentially dead rn